datasource-local/src/main/java/cmm/apps/esmorga/datasource_local/user/model/UserLocalModel.kt

```diff
@@ -0,0 +1,12 @@
+package cmm.apps.esmorga.datasource_local.user.model
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity
+data class UserLocalModel(
+    @PrimaryKey(autoGenerate = true) val id: Int = 0,
```

Comment:
email could be the PrimaryKey, no need to have a new field for that. I think this can create issues if you use try to insert a second user.